content-src/components/DiscoveryStreamComponents/TopSites/SearchShortcutsForm.jsx

```diff
@@ -0,0 +1,141 @@
+import {actionCreators as ac, actionTypes as at} from "common/Actions.jsm";
+import {FormattedMessage} from "react-intl";
+import React from "react";
+import {TOP_SITES_SOURCE} from "./TopSitesConstants";
```

Comment:
I kinda just copied all of topsites over, I wouldn't expect this to be the final solution, but it did kinda just work.

I looked at the code a bit, and we'll probably need to decide on what to do with some of the features in the topsite components, like drag and drop, and what's the value/harm in keeping it. (the whole topsites section was more complex than I would have guessed)